src/ptpd.h

```diff
@@ -44,7 +44,8 @@
 #define PTPD_H_
 
 #ifdef HAVE_CONFIG_H
-# include <config.h>
+// # include <config.h>
+# include <config_cmake.h>
```

Comment:
Rename `config_cmake.h` to `config.h` when finished.

Probably remove the `HAVE_CONFIG_H` macro altogether.